config/default.yml

```diff
@@ -141,6 +141,10 @@ RSpec/RepeatedDescription:
   Enabled: true
   Description: Check for repeated description strings in example groups.
 
+RSpec/RepeatedExample:
+  Enabled: true
+  Description: Detect repeated examples within example groups.
```

Comment:
Minor point, but should we follow the "Check..." sentence convention used by the other cops?